tests/Unit/StateStorageHealthCheckTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright 2024 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Unit;
+use OpenConext\MonitorBundle\HealthCheck\HealthReportInterface;
+use OpenConext\MonitorBundle\Value\HealthReport;
+use PHPUnit\Framework\TestCase;
+use Surfnet\Tiqr\HealthCheck\StateStorageHealthCheck;
+use Surfnet\Tiqr\Tiqr\TiqrServiceInterface;
+
+class StateStorageHealthCheckTest extends TestCase
```

Comment:
Thanks for adding the tests for the health checkers. Please update the tests with the logging implementation that I suggested above. Or if you'd like to stich with the `&$message` solution. Then include that feature in test too